src/apps/service/apps.clj

```diff
@@ -285,8 +285,9 @@
   (jobs/get-job-relaunch-info (get-apps-client user) user job-id))
 
 (defn stop-job
-  [user job-id]
-  (jobs/stop-job (get-apps-client user) user job-id)
+  [user job-id params]
+  (let [status (:job_status params "Canceled")]
```

Comment:
note to self, this should probably use the names in the persistence namespace (maybe the enum above too)